src/assets/sass/protocol/base/elements/_forms.scss

```diff
@@ -0,0 +1,87 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../includes/lib';
+
+label {
+    display: block;
+    font-weight: bold;
+    margin: 0 0 .25em;
+    padding: 0 $spacing-sm;
+
+    &.mzp-u-inline {
+        @include bidi(((padding, 0 $spacing-sm 0 0, 0 0 0 $spacing-sm),));
+        display: inline;
+        margin: 0;
+    }
+}
+
+button,
+input,
+select,
+textarea {
+    @include open-sans;
+    @include text-body-md;
+}
+
+textarea {
+    height: auto;
+    max-width: 100%;
+}
+
+textarea,
+input[type='email'],
+input[type='url'],
+input[type='tel'],
+input[type='password'],
+input[type='search'],
+input[type='text'],
+input[type='number'],
+input[type='date'],
+input[type='time'] {
+    @include border-box;
+    background: #fff;
```

Comment:
Nit: `color-secondary`?

Reply:
I'm unsure about the primary and secondary token names but that's another bike shed to paint. I'll use them for now.